lib/components/ActionTrigger/ActionTrigger.tsx

```diff
@@ -54,14 +56,19 @@ export const ActionTrigger = React.memo((props: ActionTriggerProps) => {
         <Attr.div
             className={className}
             attr={props.attr?.container}>
-            <Icon
-                icon={props.icon}
-                className={css('action-trigger-icon')}
-                labelClassName={css('action-trigger-label')}
-                size={IconSize.xsmall}
-                attr={props.attr?.icon}>
-                {props.label}
-            </Icon>
+            {typeof props.icon === 'string' 
+                ? <Icon
+                    icon={props.icon}
+                    className={iconClassName}
+                    labelClassName={labelClassName}
+                    size={IconSize.xsmall}
+                    attr={props.attr?.icon}>
+                    {props.label}
+                </Icon>
+                : <>
+                    <span className={iconClassName}>{props.icon}</span>
```

Comment:
Is this the same DOM structure that the `Icon` component yields?

Reply:
not quite, the `Icon` component yields: 
```
<div class='action-trigger-container'>
<span>
::before
<span>{label}</span>
</span>
</div>
```
while the new structure is:
```
<div class='action-trigger-container'>
<span>{label}</span>
<span>{icon}</span>
</div>
```